public/app/plugins/panel/table/column_options.ts

```diff
@@ -47,6 +48,13 @@ export class ColumnOptionsCtrl {
       { text: 'Value to text', value: 1 },
       { text: 'Range to text', value: 2 },
     ];
+    this.alignTypes = [
+      { text: 'default', value: '' },
+      { text: 'left', value: 'left' },
+      { text: 'center', value: 'center' },
+      { text: 'right', value: 'right' },
+      { text: 'justify', value: 'justify' },
+    ];
```

Comment:
Can we make these: auto, center, left, right  (and leave out justify for now)

Reply:
`auto` being `default` I guess?